.github/workflows/build.yml

```diff
@@ -1,16 +1,14 @@
-name: Spotless and Build
+name: Publish Release
```

Comment:
Both the name of the file and this name here is conflicting, and neither seem to be good. 

This file should be named `push.yml`. The name should probably be `On push`.